conda_smithy/feedstock_content/circle.yml

```diff
@@ -6,8 +6,7 @@ dependencies:
   # Note, we used to use the naive caching of docker images, but found that it was quicker
   # just to pull each time. #rollondockercaching
   override:
-    - docker pull pelson/obvious-ci:latest_x64
-#    - docker pull pelson/conda32_obvious_ci
+    - docker pull condaforge/linux-anvil
```

Comment:
In retrospect, we should have changed this to `{{ docker.image }}`, but it appears this will be resolved with PR ( https://github.com/conda-forge/conda-smithy/pull/551 ).